src/index.js

```diff
@@ -0,0 +1,74 @@
+import 'lodash';
+import './style.css';
+import sync from './assets/icons/sync.png';
+import downLeft from './assets/icons/down_left.png';
+import menuIcon from './assets/icons/menuIcon.png';
+
+const todoContaiter = document.querySelector('.todoContaiter');
+const titleContainer = document.querySelector('.title-list');
+const descriptionContainer = document.querySelector('.description-list');
+
+const tasksArray = [
+  {
+    index: '3',
+    description: 'first task',
+    completed: false,
+  },
+  {
+    index: '1',
+    description: 'second task',
+    completed: false,
+  },
+  {
+    index: '2',
+    description: 'third task',
+    completed: false,
+  },
+];
+
+// sort by index
+tasksArray.sort((a, b) => a.index - b.index);
```

Comment:
- [ ] [OPTIONAL] Kindly move the logic that sorts the `tasksArray` in the first line inside the method that is responsible for rendering tasks `mainPage` in order to be called each time you need to render the list. You will need this in upcoming projects.